addons/website_forum/views/website_forum.xml

```diff
@@ -651,13 +670,23 @@
                         </div>
                         <a t-att-href="url_for('/forum/') + slug(forum) + '?' + keep_query( 'search', 'sorting', 'filters', 'my')" class="btn py-1">&#215;</a>
                     </span>
+                    <span t-if="author_id" t-attf-class="w-100 w-md-auto mb-2 mb-md-0 border rounded pl-2 d-inline-flex align-items-center justify-content-between #{search and 'ml-md-2'}">
+                        <div>
+                            <t t-set="author" t-value="env['res.users'].browse(author_id)"/>
```

Comment:
can you give it directly to the template ? (replace author_id by author before rendering the template)